.github/CHANGELOG.md

```diff
@@ -2,6 +2,45 @@
 
 <h3>New features since last release</h3>
 
+<h4>PennyLane now comes packaged with PennyLane-Lightning</h4>
+
+* The C++-based [PennyLane-Lightning](https://pennylane-lightning.readthedocs.io/en/latest/) device
+  is now included with installations of PennyLane. The PennyLane-Lightning device is a fast
+  state-vector simulator equipped with the efficient
+  [adjoint method](https://arxiv.org/abs/2009.02823) for differentiating quantum circuits. The
+  device can be accessed using the `"lightning.qubit"` name:
```

Comment:
minor question -- do we want to market it using mainly the `PennyLane-Lightning` name, or the `lightning.qubit` name?

I was always under the assumption that `PennyLane-Lightning` was somewhat temporary, and just created to keep the code in a separate repo until integrated into PennyLane proper. Whereas `lightning.qubit` is part of the UI.

Reply:
Very very minor, I don't mind either way!

Reply:
Nice idea, have updated: 22bbe8cdb1b0b5bd77ab7c86b335571b86b68490